### DISCUSSION TITLE
mmap配置问题

### DISCUSSION THREAD

**Discussion by deict** [milvus-io/milvus#33282] [Q&A and General discussion]
根据https://milvus.io/docs/mmap.md#MMap-enabled-Data-Storage配置了mmap，指定了mmap文件夹，但是批量写了一批数据，mmap文件夹是空的，我想问下是不是配置的不对，没有起到作用？

**Comment by yhmo** [comment]
Create an index, load the collection and do some search. You will see some sub-directories created under the mmap path.

**Reply by DeerEyre** [reply]
您好，子目录的内容也是空的，同时子目录的占用空间为0KB，这正常吗

**Reply by xiaofan-luan** [reply]
目录已经被unlink了，操作系统是看不到milvus的占用的
